test/cases/ostree-vsphere.sh

```diff
@@ -0,0 +1,787 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh none
+
+# Get OS data.
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
+
+source /usr/libexec/tests/osbuild-composer/shared_lib.sh
+
+# Install govc
+sudo curl -L -o - "https://github.com/vmware/govmomi/releases/download/v0.30.5/govc_Linux_x86_64.tar.gz" | sudo tar -C /usr/local/bin -xvzf - govc
```

Comment:
Please define a GOVC_VERSION at the start of this script. Any later update, we just need update this varible.

Reply:
Fixed.